llpc/context/llpcCompiler.cpp

```diff
@@ -158,6 +159,10 @@ opt<int> ContextReuseLimit("context-reuse-limit",
 // -fatal-llvm-errors: Make all LLVM errors fatal
 opt<bool> FatalLlvmErrors("fatal-llvm-errors", cl::desc("Make all LLVM errors fatal"), init(false));
 
+// -add-hash-to-elf
+opt<bool> AddHashToELF("add-hash-to-elf", llvm::cl::desc("Add notes to ELF for hash and llpc version"),
```

Comment:
The same as above. Remove all llvm:: prefix in .cpp source file.

Reply:
done